hw19/js/script.js

```diff
@@ -0,0 +1,232 @@
+class Human {
+  constructor(info) {
+    this.name = info.name;
+    this.surname = info.surname;
+    this.age = info.age;
+  }
+  getFullName() {
+    return `${this.name} ${this.surname}`;
+  }
+  setFullName(fullName) {
+    [this.name, this.surname] = fullName.split(" ");
+  }
+}
+
+let human = new Human({ name: "Elon", surname: "Mask", age: 51 });
+
+console.log(human);
+const getName = human.getFullName();
+human.setFullName("Tom Cruise");
+
+console.log(`Get full name: ${getName}`);
+console.log(`Setted name: ${human.name} \nSetted surname: ${human.surname}`);
+console.log(human);
+
+class Student extends Human {
+  constructor({ name, surname, age, marks }) {
+    super({ name, surname, age });
+    this.marks = marks;
+  }
+
+  getAverageMark() {
+    let summaOfMarks = 0;
+    this.marks.forEach((item) => {
+      summaOfMarks += item;
+    });
```

Comment:
you can use reduce method instead of 
```javascript
let summaOfMarks = 0;
this.marks.forEach((item) => {
   summaOfMarks += item;
});
```
```javascript
const sumOfMarks = this.marks.reduce((a, b) => a + b);
```